### ISSUE TITLE
[Question] How can I exit the tmux session or switch to the parent shell?

### ISSUE THREAD

**Issue by tracure1337** [zpm-zsh/tmux#5]
[Question] How can I exit the tmux session or switch to the parent shell?

This is a very useful plugin but sometimes I need to switch to the parent shell for doing something and currently what I do is close the terminal and comment out the plugin and restart to get into a non-tmux shell.

How can I bind this to a command?   just typing exit closes my terminal  emulator as well.

Thanks in advance

**Comment by grigorii-horos** [comment]
Hey @cocoonkid. Maybe do you want to disable tmux autostart? You can set TMUX_AUTOSTART=true in `~/.zshrc`
You will keep all features from this plugin. But you will need to start tmux manually.

**Comment by tracure1337** [comment]
Yes, that's what I am currently doing. Thank you for taking the time to reply.
